packages/vite/src/node/utils.ts

```diff
@@ -1222,6 +1234,10 @@ export function joinUrlSegments(a: string, b: string): string {
   return a + b
 }
 
+export function initialSlashRemoved(str: string): string {
```

Comment:
nit: i think it's nicer if the function is named verb+action to describe what it's doing, instead of the result. so maybe `removeLeadingSlash` 😬

Reply:
haha, I like yours 😬
No need for grimacing, never married to names. One thing I was thinking here was in the [change array by copy proposal](https://github.com/tc39/proposal-change-array-by-copy) but I forgot they ended up adding the `to` in `toReversed`. I'll change it to `removeLeadingSlash`